config/eslint/index.js

```diff
@@ -11,19 +17,33 @@ module.exports = {
     },
   },
   rules: {
-    // Disabled cause it doesn't currently work as of 01/02/2021
     'no-redeclare': 'off',
     '@typescript-eslint/no-redeclare': 'off',
-    // Turned off because we use custom-classnames for our design tokens
     'tailwindcss/no-custom-classname': 'off',
     'tailwindcss/classnames-order': 'error',
   },
   parserOptions: {
     babelOptions: {
       parserOpts: {
-        // Allow imports like `import pkg from './package.json' assert { type: 'json' };`
         plugins: ['importAssertions'],
       },
     },
   },
+  ignorePatterns: ['docs/.contentlayer/generated/ContentPage/_index.mjs'],
```

Comment:
This should not be part of the shareable config, right? It's very specific to our use case

Reply:
I am so sorry, did notice or notified by this @sebald

Reply:
@OsamaAbdellateef did you already resolve this?

Reply:
@OsamaAbdellateef Is the change pushed? Didn't see it 😅

Reply:
@OsamaAbdellateef Thanks for moving it into the global eslint. Now we merge it and should have an eye on it if all rules are applied.